### PR TITLE
PD1-4567: Updated logic to allow more copy changes

### PR DESCRIPTION
**Pull request by c-emil**
https://mindojo.atlassian.net/browse/PD1-4567

Related:
* https://github.com/mindojo/live-visitors-interview/pull/4
* https://github.com/mindojo/bloombergprep/pull/73
* https://github.com/mindojo/platformized-landings/pull/14

### CODE REVIEW

File: src/embedded/main.js [hunk 1]
```diff
@@ -2,7 +2,20 @@ import Vue from 'vue';
...
 
 
-function init(course, title, websocketHostUrl, mainColor, overlayColor, logoUrl, backgroundImageUrl) {
+function init(
+  course,
+  title = 'Interview invitation',
+  titleMobile = 'Interview invitation',
+  subtitle,
+  subtitleMobile,
+  dismissButtonText = 'Don’t show this again',
```
Comment (kundojo) [inline src/embedded/main.js:11]: why dont set this as default value in component itself?

Comment (c-emil) [reply src/embedded/main.js:11]: I don't know how to set it as a default value in Vue, I didn't even think of it, can you instruct me on how to do it?

Comment (kundojo) [reply src/embedded/main.js:11]:
```
    titleMobile: {
      type: String,
      required: true,
      default: 'nyak',
    },
```

Comment (c-emil) [reply src/embedded/main.js:11]: easy :D thanks

**Review by kundojo — APPROVED** [review]

**Pull request merged**